src/task.js

```diff
@@ -0,0 +1,20 @@
+export const tasks = localStorage.getItem('tasks') ? JSON.parse(localStorage.getItem('tasks')) : [];
+
+export default class Task {
+  constructor(desc, index) {
+    this.desc = desc;
+    this.completed = false;
+    this.index = index;
+  }
+}
+
+export const addTask = (desc) => {
+  const task = new Task(desc, tasks.length + 1);
+  const newTasks = [...tasks, task];
+  localStorage.setItem('tasks', JSON.stringify(newTasks));
+};
+
+export const removeTask = (index) => {
+  const newTasks = tasks.filter((task) => task.index !== index);
+  localStorage.setItem('tasks', JSON.stringify(newTasks));
+};
```

Comment:
- Great job so far :+1: Your app has all the correct functionalities :fire: but it does not follow the UI design from the given [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). It would be great if you could address the following issues :+1: 
- [ ] First, in the given video, after a task is added or deleted, it should be done automatically without the user having to refresh the page. But if you must refresh your site to get the updated changes, kindly do that in your code and not prompt a user to do it. As a result, set it that your site automatically refreshes after a user adds and deletes an element so that the user will not have to do it manually :+1: 
- [ ] Finally, kindly follow the UI design from the video very closely so that your list will look like the given wireframe. Pay special attention to the part where a user has to delete or edit an element. When a user clicks the 3 dots icon, a delete icon should appear and the user should also be able to edit the element too. Kindly work on this issue :100: 
- Your site:
![Screenshot from 2022-08-11 06-07-32](https://user-images.githubusercontent.com/85113567/184068798-6702bda1-674f-4681-bc21-f17d4200ea2a.png)
- Wireframe:
![Screenshot from 2022-08-11 06-18-26](https://user-images.githubusercontent.com/85113567/184068877-26d76373-6b6f-463a-9c8e-5fb248ecaf23.png)

Reply:
Thank you @Kingstalux, 
I had doubts about the UI sometimes when recording the video the pop-up menus don't appear in the recordings.

Thanks for the suggestions I will implement them.

Reply:
If you have any doubts, just ask and I'll try to rectify them :+1:

Reply:
Thanks, @Kingstalux  everything was clear.